packages/components/src/components/circular-progress/circular-progress.jsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
```

Comment:
Please use `import Icon from '../icon.jsx` to avoid duplicate css imports and style duplications.

Reply:
@zaki-binary `Icon` is imported like this in all other components of deriv-components and I think this is true because it is looking at `index,js` file inside `Icon` folder by default and in that file we are importing both `jsx` and 'css'. if I import the `jsx` directly here, we miss the styling of the component.

Reply:
I think it's already bundled on export, the issue is that it caused a lot of redundancy and in some cases, as the redundancy caused unnecessary overrides as well.

You can discuss with @reza-fs  as we encountered this issue before in one of the components we added.